tests/test_entrypoints.py

```diff
@@ -1,8 +1,23 @@
 def test_hyp3_autorift(script_runner):
-    ret = script_runner.run('hyp3_autorift', '-h')
+    ret = script_runner.run('autorift', '+h')
```

Comment:
I think we need a test for `autorift` *in addition* to `hyp3_autorift`, e.g.

https://github.com/ASFHyP3/hyp3-insar-gamma/blob/develop/tests/test_entrypoints.py#L1-L9